<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageAsset_Type</class>
 <widget class="QWizardPage" name="PageAsset_Type">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="radioButtonCurrency">
     <property name="text">
      <string>Currency</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonSecurity">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Security</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonBasket">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Basket</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
